src/Angor/Client/Pages/Invest.razor

```diff
@@ -692,7 +697,7 @@ else
             long minInvestmentAmount = Money.Satoshis(0.001m) ; // Minimum investment for all networks
             long investmentAmount = Investment.InvestmentAmountBtc.ToUnitSatoshi();
 
-            if (investmentAmount < minInvestmentAmount)
+            if (Investment.InvestmentAmountBtc < minInvestmentAmount)
```

Comment:
I notice the bottom param might also be wrong `investmentAmount > maxInvestmentAmount`

Reply:
`maxInvestmentAmount` and `minInvestmentAmount` should be in btc and I think they are in sats.
not big deal I can fix it after this is pushed

Reply:
fixed
```
            bool isTestnet = network.NetworkType == NetworkType.Testnet;
            decimal maxInvestmentBtc = isTestnet ? decimal.MaxValue : 0.1m; // 0.1 BTC for mainnet, unlimited for testnet
            decimal minInvestmentBtc = 0.001m; // Minimum investment for all networks
```